### DISCUSSION TITLE
Which is the latest stable version?

### DISCUSSION THREAD

**Discussion by Dark-Noir** [trevorsandy/lpub3d#711] [General]
Hey,

i tried to create instructions for my MOC with, 2.4.6.144.3255_20230412, but it crashes and throws errors everywhere.
Then i tried 2.4.6.0.3111_20230111 which works better, but still have issues.

Which is the latest stable version for Windows?

Edit: I tested now over five different versions and every one has bugs of some sort, on the most, i can't change the edge line width and i get many random errors and crashes, on some, my sub models don't get rendered correctly (just a yellow cube), on every, i can't change the part list annotation style (square, circle etc.).

I hoped for a good alternative to the Studio Instruction Maker, but this one, i just can't get to work correctly.

**Comment by d-j-m-0** [comment]
I'm using Release 2.4.6 Revision 294 build 3405 i.e. more recent than the version you appear to be using. It seems reasonably stable so far.
